src/helper.js

```diff
@@ -0,0 +1 @@
+export const days = ["sun", "mon", "tue", "wed", "thu", "fri", "sat"];
```

Comment:
this is starting to get used more and more so why not put it as a constant in this helper

Reply:
I haven't used helpers much, I'm interested in learning more about this.

Reply:
https://codeigniter.com/user_guide/general/helpers.html